ruby/lesson3/tutorial.md

```diff
@@ -90,7 +90,7 @@ If you haven't started going through Ruby Koans yet, you can find more informati
 
 ## More practise
 
-There are a lot of online interactive tutorials that you can work through. Understanding basic concepts its important before moving forward. Try practising what you've learned so far by coming up and applying your own ideas. If you get stuck or needs help, bring your questions with you to our next session or ask your questions on our [gitter chat](gitter.im/codebar/tutorials)
+There are a lot of online interactive tutorials that you can work through. Understanding basic concepts its important before moving forward. Try practising what you've learned so far by coming up and applying your own ideas. If you get stuck or needs help, bring your questions with you to our next session or ask your questions on our [gitter chat](https://gitter.im/codebar/tutorials)
```

Comment:
We actually use Slack now not Gitter so could this whole thing be changed to point to Slack, and the URL being https://slack.codebar.io/ please?

Nice spot :)

Reply:
Sure! I think I've seen Gitter linked somewhere else too. I'll change that too if I can find it :)

Thank you!